es5.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     rules: {
       'comma-dangle': 'off',
       indent: ['error', 4],
-      'no-param-reassign': ['error', { 'props': false }]
+      'no-param-reassign': ['error', { 'props': false }],
+      'object-curly-newline': ['error',  { 'multline': true, 'consistent': true }]
```

Comment:
I tried running a test with this on the ML UI and it is failing with an error. Can you link to the documentation you referenced for this setting? I am not seeing this format in the docs https://eslint.org/docs/rules/object-curly-newline